CHANGELOG.md

```diff
@@ -48,7 +48,8 @@ messages inside `authz.MsgExec`
 
 ### Improvements
 
-* #[131](https://github.com/babylonlabs-io/babylon/pull/131) Add new staking
+* [#131](https://github.com/babylonlabs-io/babylon/pull/131) and
+  [#144](https://github.com/babylonlabs-io/babylon/pull/144) Add new staking
```

Comment:
I would separte change log entries for:
- new staking events
- new finality provider events